tempesta_fw/http_parser.c

```diff
@@ -929,6 +963,26 @@ __FSM_STATE(RGen_HdrOtherV) {						\
 	__FSM_MOVE_n(RGen_EoL, __fsm_sz);				\
 }
 
+/*
+ * Same as __FSM_MOVE_nofixup, but exit with TFW_POSTPONE code if the
+ * message is processed in streaming mode. Required to split buffered part
+ * of the message (headers) from streaming (body) in cases when buffered
+ * part is too long and may overflow buffering limits.
+ */
+#define __FSM_MOVE_body_nofixup(to)					\
+do {									\
+	p += 1;								\
+	if (unlikely(msg->flags & TFW_HTTP_F_STREAM)) {			\
+		__fsm_const_state = to; /* start from state @to next time */\
+		__FSM_EXIT(TFW_SPLIT);					\
```

Comment:
It seems there is no difference in handling `TFW_SPLIT` and `TFW_POSTPONE`, so it turns out that there is no need for the macro as well.